### PR TITLE
gmcnn-places2-tf is modified for BGR->BGR processing.

### PR DESCRIPTION
**Pull request by fzhar**
Accuracy checker is updated correspondingly.

### CODE REVIEW

**Review by eaidova — APPROVED** [review]

**Comment by IRDonch** [comment]
What you have looks good, but aren't you going to update `model.yml` to use the newly re-frozen model?

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]
please pay attention to @IRDonch comment on updating model.yml (if needed, or confirm that update is not required)

**Review by vladimir-dudnik — APPROVED** [review]

File: models/public/gmcnn-places2-tf/model.yml [hunk 1]
```diff
@@ -20,8 +20,8 @@ description: >-
 task_type: image_inpainting
 files:
   - name: frozen_model.pb
-    size: 49774888
-    sha256: 2bef90eb90a11a7bf552df66e6d3c86969c922283236686ef8225dc4ee21c39d
+    size: 49774390
+    sha256: 15ac9c873c24ad8ed9bf3f37490b0dfa87a3af3dcf3cf71fd27a31a79a498d02
     source: https://download.01.org/opencv/public_models/022020/gmcnn/frozen_model.pb
```
Comment (vladimir-dudnik) [inline models/public/gmcnn-places2-tf/model.yml:25]: seems yo need to specify new path to updated model, 092020, is not it?

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins ok to test

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by fzhar** [comment]
Jenkins please retry a build

**Comment by eaidova** [comment]
Jenkins please retry a build

**Review by vladimir-dudnik — APPROVED** [review]

**Pull request merged**